src/position-detail/position-summary-controls/index.tsx

```diff
@@ -201,12 +203,54 @@ const PositionSummaryControls = ({
                 disabledWithdrawFunds ||
                 isPending ||
                 disabled ||
-                BigNumber.from(position.remainingLiquidity).lte(BigNumber.from(0)),
+                BigNumber.from(position.remainingLiquidity).lte(BigNumber.from(0)) ||
+                (isFromProtocolToken && !canWithdrawProtocolToken),
               onClick: onWithdrawFunds,
             },
+            ...(isFromProtocolToken
+              ? [
+                  {
+                    text: (
+                      <FormattedMessage
+                        description="withdrawWrapped"
+                        defaultMessage="Withdraw remaining as {wrappedProtocolToken}"
+                        values={{
+                          wrappedProtocolToken: wrappedProtocolToken.symbol,
+                        }}
+                      />
+                    ),
+                    disabled:
+                      isPending || disabled || BigNumber.from(position.remainingLiquidity).lte(BigNumber.from(0)),
+                    onClick: () => onWithdrawFunds(false),
+                  },
+                ]
+              : []),
           ]}
         />
       )}
+      {shouldDisableArrow && mergedPermissions.WITHDRAW && !mergedPermissions.REDUCE && (
```

Comment:
Why is reduce taken into account here?

Reply:
If the user has permissions to REDUCE we need the split button because they can withdraw base funds